optimism2/ovm2/get_contracts/insert_get_contracts.sql

```diff
@@ -0,0 +1,302 @@
+CREATE OR REPLACE FUNCTION ovm2.insert_get_contracts(start_blocktime timestamptz, end_blocktime timestamptz, creator_list bytea[] = NULL::bytea[]) RETURNS integer
```

Comment:
i'm getting some syntax errors when attempting to create this function. looks like the CTE's are missing closing parentheses and/or too many commas. here is a head-start on the query in dune if you want to copy/paste into one and try to get it running as expected, then push changes here:
https://dune.com/queries/835559

Reply:
oh weird yeah there were a few differences between this and what I was running on my end. Updating